CHANGELOG.md

```diff
@@ -335,9 +335,50 @@ Example for file plugin output:
 
 - pcsampler sample code has been removed due to deprecation from v2.
 
-## ROCprofiler for rocm 6.3
+## ROCProfiler for ROCm 6.3
 
 ### Added
 
-- JSON output plugin for rocprofv2, the JSON file matches Google Trace Format, so it should be easily loaded to perfetto, chrome tracing or speedscope. For Speedscope, `--disable-json-data-flows` option will be needed as speedscope doesn't work with data flows.
-- Added `--no-serialization` flag to disable kernel serialization when rocprofv2 is in counter-collection mode. This change was added to allow rocprofv2 to avoid deadlock when profiling certain programs in counter-collection mode.
\ No newline at end of file
+- JSON output plugin for `rocprofv2`. The JSON file matches Google Trace Format making it easy to load on Perfetto, Chrome tracing, or Speedscope. For Speedscope, use `--disable-json-data-flows` option as speedscope doesn't work with data flows.
+- `--no-serialization` flag to disable kernel serialization when `rocprofv2` is in counter collection mode. This allows `rocprofv2` to avoid deadlock when profiling certain programs in counter collection mode.
+-	`FP64_ACTIVE` and `ENGINE_ACTIVE`
+- New HIP APIs with struct defined inside union.
+- Early checks for ELF file
+- Support for kernel name filtering in `rocprofv2`
+- Barrier bit to read and stop packets
+- ROCProfiler support for gfx1150 and gfx1151
+- ATT support for gfx12
+- gfx12 support
+
+### Changed
+
+- Extended lifetime for proxy queues
+- Setting the `trace-start` option for `rocprof` to `off` now disables kernel tracing
+- Load `libpciaccess-dev` functions with `dlopen`
+-	Initialize `PcieAccessApi*` api and `void* libpciaccess_handle` to `nullptr`
+
+### Removed
+
+- Extra licenses
```

Comment:
@bgopesh Can we provide some more info here

Reply:
Removed some BSD and GPL licenses which did not correspond to any code.

Reply:
Done